### DISCUSSION TITLE
Jetty 12 / Eclipse 2024-09

### DISCUSSION THREAD

**Discussion by frankbenoit** [eclipse-glsp/glsp#1418] [General] [answered]
Hi

Eclipse switched to Jetty 12.0.12, are there plans to get in sync here?

Frank

**Comment by martin-fleck-at** [comment, marked as answer]
Hi Frank, yes, we are planning to support upgrades to newer Eclipse versions for the next release. I added this to the ticket that targets the Jakarta update as well, see https://github.com/eclipse-glsp/glsp/issues/1397.
